app/controllers/order/goods_details.js

```diff
@@ -0,0 +1,62 @@
+import Ember from "ember";
+const { getOwner } = Ember;
+import AjaxPromise from 'browse/utils/ajax-promise';
+
+export default Ember.Controller.extend({
+  queryParams: ["typeId", "fromClientInformation"],
+  order: Ember.computed.alias("model"),
+  typeId: null,
+  fromClientInformation: false,
+  qty: null,
+  otherDetails: "",
+  requestCount: 3,
```

Comment:
Why is this set to 3?

Reply:
This is not needed. I will remove it. Thanks 👍